lib/routes/charge/currency_wrapper.dart

```diff
@@ -48,6 +49,18 @@ class CurrencyWrapper {
     return fiat.exchangeRate / 100000000;
   }
 
+  List<TextInputFormatter> get inputFormatters {
```

Comment:
We should return here only the Regex. The TextInputFormatter is a UI element.
Also better to add it on Currency and FiatConversion and call them from here.

Reply:
[Done](https://github.com/breez/breezmobile/pull/296/commits/890d18227d192ef7b4d060055869bc53b48cca1e)